subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak_minimal.c

```diff
@@ -141,11 +141,13 @@ int fp_storage_ak_find(struct fp_account_key *account_key,
 	return -ESRCH;
 }
 
-int fp_storage_ak_save(const struct fp_account_key *account_key)
+int fp_storage_ak_save(const struct fp_account_key *account_key, const void *conn_ctx)
 {
 	int err;
 	struct account_key_data data;
 
+	__ASSERT_NO_MSG(!conn_ctx);
```

Comment:
`ARG_UNUSED` -> we pass the conn object unconditionally here, so the assertion would actually fail